openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -93,7 +94,7 @@ def _verify_full_shareable_account_response(self, response):
         self.assertIsNone(data["profile_image"])
         self.assertIsNone(data["time_zone"])
         self.assertIsNone(data["languages"])
-        self.assertIsNone(data["bio"])
+        self.assertEqual("Tired mother of twins", data["bio"])
```

Comment:
Maybe there should be a test to verify clearing of the bio and making sure that the change sticks, i.e. that empty string updates are not treated as no-ops.